bootloader/grub.go

```diff
@@ -457,3 +460,25 @@ func staticCommandLineForGrubAssetEdition(asset string, edition uint) string {
 	}
 	return string(cmdline)
 }
+
+// TrustedAssetsChain returns the list of relative paths to files inside
+// the bootloader's rootdir that are measured in the boot process in the
+// order of loading during the boot.
+func (g *grub) TrustedAssetsChain() []string {
+	if g.native {
+		if g.recovery {
+			return []string{
+				// recovery mode shim EFI binary
+				"EFI/boot/bootx64.efi",
+				// recovery mode grub EFI binary
+				"EFI/boot/grubx64.efi",
+			}
+		} else {
+			return []string{
+				// run mode grub EFI binary
+				"EFI/boot/grubx64.efi",
+			}
+		}
+	}
+	return nil
```

Comment:
shouldn't we return an internal error in this case instead? I'm also a bit confused, won't the bootloader used at reseal not be native anyway so we need to do something here/else?